src/icrar/leap-accelerate-cli/Arguments.cc

```diff
@@ -196,6 +213,16 @@ namespace icrar
         return *m_inputStream;
     }
 
+    boost::optional<std::string> ArgumentsValidated::GetOutputFilePath() const
+    {
+        return m_outputFilePath;
+    }
+
+    std::ostream& ArgumentsValidated::GetOutputStream()
+    {
+        return *m_outputStream;
```

Comment:
`return *m_outputFileStream.get()`

Reply:
I store m_outputStream seperately since in some cases m_outputStream is set to std::cout

Reply:
Ahhhh right, my bad, I didn't see that the first time. All good then :)

Reply:
Thinking about it again, an alternative would be to only store `m_outputFileStream`, then implement `GetOutputStream()` like:

```c++
std::ostream &ArgumentsValidated::GetOutputStream()
{
    if (m_outputFileStream)
    {
        return *m_outputFileStream.get();
    }
    return std::cerr;
}
```

Reply:
Pretty much, the only difference between the two is m_outputStream is acting as a cache variable